lib/popcorn/tests/Makefile

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013 Alexandre Terrasa <alexandre@moz-code.org>.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+all: tests
+
+tests: clean
```

Comment:
for some [random reasons](https://www.gnu.org/prep/standards/html_node/Standard-Targets.html), the test target is named `check`. Jenkins will execute them if the makefile with the check target has to be in the main directory (i.e. `lib/popcorn/Makefile`)